src/styles/styles.scss

```diff
@@ -1,2 +1,3 @@
 @forward "helpers";
 @forward "variables";
+@forward "transitions";
```

Comment:
```suggestion
```
You have `@use "../../../styles/transitions";` in components. Please remove it from this file. Because you duble transitions